src/store/modules/job/actions.ts

```diff
@@ -248,6 +248,58 @@ const actions: ActionTree<JobState, RootState> = {
       showToast(translate("Something went wrong"));
     })
   },
+  async fetchMiscellaneousJobs({ commit, dispatch, state }, payload){
+    const params = {
+      "inputFields": {
+        "enumTypeId": "MISC_SYS_JOB",
+        "statusId": ["SERVICE_PENDING", "SERVICE_DRAFT"],
+        "statusId_op": "in",
+        "systemJobEnumId_op": "not-empty"
+      } as any,
+      "fieldList": [ "systemJobEnumId", "runTime", "tempExprId", "parentJobId", "serviceName", "jobId", "jobName", "currentRetryCount", "statusId", "enumName" ],
```

Comment:
Add description and shopId in fieldList.

Reply:
Added sir.